gui/wxpython/lmgr/frame.py

```diff
@@ -707,9 +689,9 @@ def CanClosePage(self, caption):
         """
```

Comment:
Looking more into this method, would you agree this should go into the manager?

Reply:
Yes, I agree, but there is a bit problem now with the OnCloseWindow method in mapdisp frame.py. I suggest letting the ugly `self._layerManager.workspace_manager.CanClosePage` solution and in the next PR I would move the majority of this 
`OnCloseWindow` method to lmgr. Do you agree with that?

Reply:
I agree, although I am not sure about moving the method to lmgr, that might be tricky. I don't like the method name CanClosePage but I don't have better idea at this moment.